py3status/modules/keyboard_locks.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+"""
+Monitor CapsLock, NumLock, and ScrLock keys
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module (default 1)
+    format: display format for this module (default '{caps} {num} {scr}')
+    icon_capslock_off: show when Caps Lock is off (default 'CAP')
+    icon_capslock_on: show when Caps Lock is on (default 'CAP')
+    icon_numlock_off: show when Num Lock is off (default 'NUM')
+    icon_numlock_on: show when Num Lock is off (default 'NUM')
+    icon_scrlock_off: show when Scroll Lock is off (default 'SCR')
+    icon_scrlock_on: show when Scroll Lock is on (default 'SCR')
```

Comment:
I think these should match the placeholders better eg `icon_scr_on` if placeholder is `{scr}`